worlds/raft/locations.json

```diff
@@ -1,874 +1,1064 @@
 [
     {
         "id": 48001,
-        "name": "Small trophy board",
+        "name": "Honey",
         "region": "ResearchTable",
         "requiresAccessToItems": [
-            "Plank",
-            "Nail"
+            "HoneyComb",
+            "Glass"
         ]
     },
     {
         "id": 48002,
-        "name": "Medium trophy board",
+        "name": "Flippers",
         "region": "ResearchTable",
         "requiresAccessToItems": [
-            "Plank",
-            "Nail"
+            "Plastic",
+            "SeaVine",
+            "VineGoo",
+            "Rope"
         ]
     },
     {
         "id": 48003,
-        "name": "Simple collection net",
+        "name": "Birds nest",
         "region": "ResearchTable",
         "requiresAccessToItems": [
+            "Thatch",
             "Plank",
-            "Rope",
-            "Nail"
+            "Placeable_GiantClam"
         ]
     },
     {
         "id": 48004,
-        "name": "Advanced grill",
+        "name": "Battery",
         "region": "ResearchTable",
         "requiresAccessToItems": [
-            "Plank",
-            "MetalIngot",
-            "Rope",
-            "Nail"
+            "CopperIngot",
+            "Scrap",
+            "Plastic"
         ]
     },
     {
         "id": 48005,
-        "name": "Large crop plot",
+        "name": "Circuit board",
         "region": "ResearchTable",
         "requiresAccessToItems": [
-            "Plank",
-            "Rope",
-            "Nail",
-            "Hinge"
+            "Plastic",
+            "CopperIngot",
+            "VineGoo"
         ]
     },
     {
         "id": 48006,
-        "name": "Scarecrow",
+        "name": "Bolt",
         "region": "ResearchTable",
         "requiresAccessToItems": [
-            "Plank",
-            "Thatch",
-            "Nail",
-            "Plastic"
+            "MetalIngot"
         ]
     },
     {
         "id": 48007,
-        "name": "Hammock",
+        "name": "Hinge",
         "region": "ResearchTable",
         "requiresAccessToItems": [
-            "Plank",
-            "Thatch",
-            "Feather",
-            "Nail"
+            "MetalIngot"
         ]
     },
     {
         "id": 48008,
-        "name": "Smelter",
+        "name": "Shear",
```

Comment:
The item name was changed from Shear to Shears, but the location is still just Shear, is this intentional?

Reply:
Good catch. I think this got caught up in the mix of updating for the Final Chapter. Reverting to Shear to stay consistent with previous iterations.